src/complex_fluids/CFRelaxationOperator.cpp

```diff
@@ -47,18 +47,14 @@ CFRelaxationOperator::convertToConformation(const MatrixNd& mat)
     {
     case SQUARE_ROOT:
         return mat * mat;
-        break;
```

Comment:
Does clang really complain about this? I would rather keep these `break` statements.

Reply:
It does when very pedantic warning flags are on.

Reply:
Yuck.

Reply:
Would it be better to capture the output and have a single `return` statement?

Reply:
The proposed implementation is the simplest - we just should immediately return rather than setting up temporary variables.

Nevertheless, I'm mostly interested in getting rid of the warning, so if that is your preference then lets do it.

Reply:
Let's leave it as is and get rid of the `break` statements to appease clang.